src/evergreen/cli/main.py

```diff
@@ -410,6 +410,26 @@ def user_permissions(ctx, user_id):
     click.echo(fmt_output(fmt, [p.json for p in permissions]))
 
 
+@cli.command()
+@click.pass_context
+@click.option("--user-id", required=True, help="User whose permissions to remove")
+@click.option(
+    "--resource-type",
+    required=True,
+    type=click.Choice(["project", "distro", "superuser", "all"], case_sensitive=False),
+    help="Type of resource for which to delete permissions.",
+)
+def delete_user_permissions(ctx, user_id, resource_type):
+    """Delete all permissions of a given type for a user."""
+    api = ctx.obj["api"]
+    fmt = ctx.obj["format"]
+
+    api.delete_user_permissions(user_id, resource_type)
+    click.echo(
+        fmt_output(fmt, f"Sucessfully deleted {resource_type} permissions for user {user_id}")
```

Comment:
[nit] The `fmt_output` function is meant to formatting data from the API in different formats, specified by commandline args (like json or yaml). Since here the output is just a string, it probably isn't needed.